lib/main_common.pm

```diff
@@ -585,6 +588,7 @@ sub load_jeos_tests {
     loadtest 'qa_automation/patch_and_reboot' if is_updates_tests;
     replace_opensuse_repos_tests if is_repo_replacement_required;
     loadtest 'console/verify_efi_mok' if get_var 'CHECK_MOK_IMPORT';
+    loadtest "console/zypper_ref" if get_var('CONTAINER_RUNTIME');
```

Comment:
Maybe we can skip this at all... not sure.
Anyway, you'll need to verify that it also works on openSUSE.

Reply:
https://openqa.opensuse.org/tests/2097111
https://openqa.opensuse.org/tests/2097112
https://openqa.opensuse.org/tests/2097114

Reply:
zypper_ref is executed twice... one from this line and the other where does it come from?

Reply:
*this change is ready for review* reason is that when o3 `replace_opensuse_repos_tests` is triggered